frontend/components/commons/forms/questions-form/fields/multi-label/MultiLabel.component.vue

```diff
@@ -0,0 +1,125 @@
+<template>
+  <div class="wrapper">
+    <div class="title-area --body2">
+      <span v-text="title" v-optional-field="isRequired ? false : true" />
+
+      <BaseIconWithBadge
+        class="icon-info"
+        v-if="!!tooltipMessage"
+        icon="info"
+        :id="`${title}Rating`"
+        :show-badge="false"
+        iconColor="#acacac"
+        badge-vertical-position="top"
+        badge-horizontal-position="right"
+        badge-border-color="white"
+        v-tooltip="{ content: tooltipMessage, backgroundColor: '#FFF' }"
+      />
+    </div>
+
+    <MultiLabelSelectionComponent
+      :options="uniqueOptions"
+      :componentId="questionId"
+      :showSearch="showSearch"
+      :multiple="true"
```

Comment:
I don't see this property defined in `MultiLabelSelectionoComponent`. Are we using int for something?